app/assets/images/payment_icons/postfinance_efinance.svg

```diff
@@ -0,0 +1 @@
4.7-.7.7zm0 .5c.8 0 1.3-.5 1.4-1.2.1-.7-.3-1.2-1.1-1.2s-1.3.5-1.4 1.2c-.1.7.3 1.2 1.1 1.2zm2.6-.7c0 .2-.2.2-.4.2-.3 0-.5-.1-.7-.1l-.1.5c.2.1.5.1.8.1.6 0 1.1-.2 1.1-.8 0-.7-.9-.6-.9-.9 0-.1.1-.2.4-.2.2 0 .5.1.6.1l.1-.5c-.2-.1-.5-.1-.7-.1-.7 0-1.1.3-1.1.8 0 .7.9.6.9.9zm2.7-1.1l.1-.5H14l.1-.6-.7.2-.5 2c-.1.6.2.9.8.9.1 0 .3 0 .5-.1l.1-.5c-.1 0-.3.1-.4.1-.2 0-.3-.1-.3-.3l.2-1h.8zm.7 1.8l.3-1.2h1.1l.1-.6h-1.1l.2-.7H17l.1-.6h-1.8l-.7 3.1h.7zm2-2.3l-.5 2.3h.7l.5-2.3h-.7zm.1-.7c0 .2.1.4.4.4.2 0 .4-.2.4-.4s-.1-.4-.4-.4c-.2.1-.4.2-.4.4zm1.2 3l.3-1.2c.1-.5.4-.6.6-.6.3 0 .4.2.3.5l-.3 1.3h.7l.3-1.3c.2-.7-.2-1-.7-1-.3 0-.6.1-.8.3l.1-.3h-.7l-.5 2.3h.7zm3.5-2.4c-.3 0-.7.1-.8.1l-.1.5c.2 0 .5-.1.7-.1.4 0 .5.1.5.3v.1H22c-.7 0-1.3.3-1.3.9 0 .4.3.6.7.6.3 0 .6-.1.7-.3l-.1.2h.6l.3-1.3v-.3c.2-.3 0-.7-.8-.7zm-.7 1.7c0-.2.2-.4.6-.4h.4v.1c-.1.3-.3.5-.6.5-.3.1-.4 0-.4-.2zm2.5.7l.3-1.2c.1-.5.4-.6.6-.6.3 0 .4.2.3.5l-.3 1.3h.7l.3-1.3c.2-.7-.2-1-.7-1-.3 0-.6.1-.8.3l.1-.3h-.7l-.5 2.3h.7zm3.4-.5c-.3 0-.5-.1-.5-.5s.3-.8.8-.8c.1 0 .3 0 .5.1l.1-.5c-.2-.1-.4-.1-.6-.1-.8 0-1.4.5-1.4 1.4 0 .7.4 1 1.1 1 .2 0 .4 0 .5-.1l.1-.5c-.2-.1-.4 0-.6 0zm2.7.4l.1-.6c-.2.1-.4.1-.7.1-.3 0-.6-.1-.6-.5h1.6c.1-.2.1-.4.1-.5 0-.5-.3-.9-1-.9-.9 0-1.4.7-1.4 1.4 0 .6.5 1 1.2 1 .3.1.5.1.7 0zm-.5-1.8c.2 0 .4.1.4.4v.1h-1c.1-.3.3-.5.6-.5zm-14.9 6.9h2l-.1.5h-1.3l-.2.7h1.2l-.2.5h-1.2l-.2.9H16l-.1.5h-2l.7-3.1zm2.2 1.7H18l-.1.4h-1.3l.2-.4zm2.3-1.7h1.8l-.1.5h-1.2l-.2.8h1.1l-.1.5h-1.1L19 20h-.7l.8-3.1zm2.1.8h.6l-.4 2.3h-.6l.4-2.3zm.3-1h.6l-.1.6h-.6l.1-.6zm1 1.3v-.3h.6v.2c.1-.1.2-.1.3-.2.1-.1.2-.1.4-.1s.4.1.6.2c.1.1.2.3.2.5v.3c0 .1 0 .2-.1.2l-.2 1.2h-.6l.2-1.1v-.4c0-.1 0-.2-.1-.2-.1-.1-.1-.1-.3-.1-.1 0-.2 0-.3.1-.1 0-.1.1-.2.2s-.1.1-.1.2-.1.2-.1.3l-.1 1H22l.5-2zm4 1.7c-.1.1-.2.2-.4.3-.1.1-.3.1-.4.1-.2 0-.4-.1-.5-.2-.1-.1-.2-.3-.2-.5s.1-.3.2-.5c.1-.1.2-.2.4-.2.2-.1.3-.1.5-.1h.7v-.3l-.1-.1h-.8c-.1 0-.3.1-.4.1l.1-.5c.1 0 .3-.1.4-.1h.8c.1 0 .2.1.3.1.1.1.2.1.2.2.1.1.1.2.1.3v.5c0 .1 0 .2-.1.3 0 .1 0 .2-.1.3 0 .1 0 .2-.1.3v.3h-.5l-.1-.3zm-.9-.3c0 .1 0 .1.1.2.1 0 .1.1.2.1s.2 0 .3-.1c.1 0 .1-.1.2-.1.1-.1.1-.1.1-.2s.1-.2.1-.3h-.5c-.1 0-.2 0-.3.1-.1 0-.2.1-.2.1v.2zM28 18v-.3h.6v.2c.1-.1.2-.1.3-.2.1-.1.2-.1.4-.1s.4.1.6.2c.1.1.2.3.2.5v.3c0 .1 0 .2-.1.2l-.1 1.2h-.6l.2-1.1v-.4c0-.1 0-.2-.1-.2-.1-.1-.1-.1-.3-.1-.1 0-.2 0-.3.1-.1 0-.1.1-.2.2s-.1.1-.1.2-.1.2-.1.3l-.2 1h-.6l.4-2zm4.5.2c-.1-.1-.3-.1-.5-.1-.1 0-.2 0-.3.1-.1.1-.2.1-.3.2-.1.1-.1.2-.2.3 0 .1-.1.2-.1.3v.2c0 .1 0 .1.1.2 0 .1.1.1.2.1s.2.1.3.1h.3c.1 0 .2 0 .3-.1v.5h-.6c-.2 0-.3 0-.4-.1-.1 0-.2-.1-.3-.2-.1-.1-.2-.2-.2-.3 0-.1-.1-.3-.1-.4 0-.2 0-.4.1-.6.1-.2.2-.3.3-.5.1-.1.3-.2.5-.3.2-.1.4-.1.6-.1h.3c.1 0 .2 0 .3.1l-.3.6zm.8.8c0 .2.1.4.2.5.1.1.3.1.5.1h.4c.1 0 .2-.1.4-.1l-.1.5c-.1 0-.3.1-.4.1h-.4c-.2 0-.3 0-.5-.1-.1 0-.3-.1-.4-.2-.1-.1-.2-.2-.2-.3-.1-.1-.1-.3-.1-.4 0-.2 0-.4.1-.5.1-.2.2-.3.3-.5.1-.1.3-.2.4-.3.2-.1.4-.1.6-.1.2 0 .3 0 .4.1.1 0 .2.1.3.2.1.1.2.2.2.3 0 .1.1.2.1.4v.6h-1.8zm1.1-.4v-.2c0-.1 0-.2-.1-.3-.1-.1-.2-.1-.3-.1-.2 0-.3.1-.4.2-.1.1-.2.2-.2.4h1z" fill="#010101"/></svg>
```

Comment:
1. The border radius look different from other icons: [video demonstration](https://screenshot.click/10-09-jxmet-bhuom.mp4)
2. I noticed that some styles are using `style=""` while some other paths are using attributes. Since all other icons are using attributes, can we follow the same thing?

[![alt](https://screenshot.click/10-14-og1ib-u5uyd.png)](https://screenshot.click/10-14-og1ib-u5uyd.png)

Reply:
refactored the styles to attributes and adjusted the borders, see: 28efe2c997fe2b35e49aec75a5644ea39c39d2cf